new-website/deepchem/components/Footer/Footer.js

```diff
@@ -9,7 +9,7 @@ import Logo from "../Logo/Logo";
  */
 function Footer() {
   return (
-    <footer className="px-[25px] 2xl:px-[300px] bg-dc-gray py-8 flex flex-col text-dc-light-gray">
+    <footer className="px-[25px] 2xl:px-[300px] bg-dc-gray py-8 flex flex-col text-dc-light-gray mt-auto">
```

Comment:
Can you explain what this change does?

Reply:
We had to stick the footer to the bottom of the screen, even when there was no content to display. To do that, there are a few options, one of them is to set the margin-top property of the footer to auto and make the body properties as flex, flex column and full view height. This makes the necessary change.